pkg/macports/ports/sysutils/salt/Portfile

```diff
@@ -1,138 +0,0 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
Is it intentional that this file is being removed, but not replaced? I didn't see an explanation in the PR description for why this is being removed, but I may have missed that.

Reply:
I'm maintaining the Portfile in the downstream macports-ports repo.
https://github.com/macports/macports-ports/tree/master/sysutils/salt

It seems silly to maintain the same file in two places with two different PR merge processes.